### ISSUE TITLE
Changing Source While In Full-Screen Does Not Preserve Scaling

### ISSUE THREAD

**Issue by martindale** [Selz/plyr#120]
When using the following example code, full-screening the video (if it's the item that plays, see #119) will correctly scale the video to full height and width, but on source rotation (every five seconds) the full-screen mode is preserved, but the video reduces down to a smaller size, leaving the rest of the screen black and unused.

```jade
extends layouts/media

block content
  h1 yo
  
  .player
    video

block scripts
  script.
    //var player = plyr.setup();
    plyr.setup();

    var player = document.getElementsByClassName('player')[0].plyr;

    var sources = [
      {
        type: 'audio',
        sources: [{
          src: 'http://inumedia.io/whalestep.mp3',
          type: 'audio/mp3'
        }],
        autoplay: true
      },
      {
        type: 'youtube',
        sources: 'SZO5QYP-GeQ',
        autoplay: true
      }
    ];

    setInterval(function() {
      player.source(sources[Math.floor(Math.random() * 2)]);
    }, 5000);

    (function(d, p){
    var a = new XMLHttpRequest(),
        b = d.body;
    a.open("GET", p, true);
    a.send();
    a.onload = function(){
        var c = d.createElement("div");
        c.style.display = "none";
        c.innerHTML = a.responseText;
        b.insertBefore(c, b.childNodes[0]);
    }
    })(document, "/img/sprite.svg");
```

**Comment by SamPotts** [comment]
Good call. Thanks for picking up on this.

**Comment by SamPotts** [comment]
Fixed in [v1.5.0](https://github.com/Selz/plyr/releases/tag/v1.5.0)
